.github/filters.yml

```
@@ -10,17 +10,12 @@ ingestion_server:
   - ingestion_server/**
   # Change to the CI + CD workflow should trigger complete workflow.
   - .github/workflows/ci_cd.yml
-frontend:
-  - frontend/**
-  - package.json
-  - pnpm-lock.yaml
-  - pnpm-workspace.yaml
-  - .npmrc
-  - .pnpmfile.cjs
-  # Change to the CI + CD workflow should trigger complete workflow.
-  - .github/workflows/ci_cd.yml
 documentation:
   - documentation/**
+  - frontend/**.mdx
+  - frontend/.storybook
+  - frontend/nuxt.config.ts
+  - frontend/tailwind.config.js
   # Change to the CI + CD workflow should trigger complete workflow.
   - .github/workflows/ci_cd.yml
```

Drop these lines w.r.t. #1933

```suggestion
  # Change to the CI + CD workflow should trigger complete workflow.
  - .github/workflows/ci_cd.yml
```